pages/download.html

```diff
@@ -1,54 +1,63 @@
 <#include "incl/macros.html">
-<#def TITLE>Coq <#CURRENTVERSION></#def>
+<#def TITLE>Install Coq</#def>
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">The current version: Coq <#CURRENTVERSION>
+<div class="frameworklabel">The Coq platform
 </div>
 <div class="frameworkcontent">
 
-<p>For downloads (in particular to get installers for Windows and Mac OS, the PDF
-  manual, or a tarball of the sources), please go to
-  <a href="<#CURRENTVERSIONURL>">the release page on GitHub</a>.</p>
+<p>Coq has a rich ecosystem of external packages (libraries and
+plugins) that extend it and make it more powerful.  The Coq platform
+provides an easy way to install Coq and a consistent set of packages
+on Windows, macOS and many Linux distributions.</p>
+
+<p>Beginners are recommended to use one of the binary installers: we
```

Comment:
```suggestion
<p>Beginners are encouraged to use one of the binary installers: we
```

Or, perhaps better, "We encourage ..."

Reply:
One might add some reasoning here:
- Binary installers = fast (a few minutes) but less flexible (cannot easily add additional packages)
- Build from sources = slow (1 ... 5 hours) but very flexible